iset.mm

```diff
@@ -107154,6 +107200,530 @@ According to Wikipedia ("Least common multiple", 27-Aug-2020,
     VLVMWDABFECZDCAFDCWEABFKLVSVNVOXCFADVPVMVQVRVR $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+    Coprimality and Euclid's lemma
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+According to Wikipedia "Coprime integers",
+see ~ https://en.wikipedia.org/wiki/Coprime_integers (16-Aug-2020) "[[...] two
+integers a and b are said to be _relatively prime_, mutually prime, or
+_coprime_ [[...] if the only positive integer (factor) that divides both of
+them is 1.  Consequently, any prime number that divides one does not divide the
+other.  This is equivalent to their greatest common divisor (gcd) being 1.".
+In the following, we use this equivalent characterization to say that
+` A e. ZZ ` and ` B e. ZZ ` are coprime (or relatively prime) if
+` ( A gcd B ) = 1 `. The equivalence of the definitions is shown by
+~ coprmgcdb .  The negation, i.e. two integers are not coprime, can be
+expressed either by ` ( A gcd B ) =/= 1 `, see ~ ncoprmgcdne1b , or
+equivalently by ` 1 < ( A gcd B ) `, see ~ ncoprmgcdgt1b .
+
+A proof of Euclid's lemma based on coprimality is provided in ~ coprmdvds (as
+opposed to Euclid's lemma for primes).
+$)
+
```

Comment:
Do the conventions about section headers for set.mm also hold for iset.mm? If yes, then they should be applied here:  
      "The header comments should be spaced as those of Part 1, namely, with
       a blank line before and after the comment, and an indentation of two
       spaces."

Reply:
Sure, if set.mm starts following them consistently. This header is copy-pasted from set.mm without re-indenting anything.

The topic is being tracked at #2262 and I'd support having iset.mm adopt whatever set.mm ends up doing.

Reply:
Yes, this is also the case with https://github.com/metamath/set.mm/pull/2436#discussion_r787220429